ingestion-edge/ingestion_edge/publish.py

```diff
@@ -39,8 +39,12 @@ def handle_request(topic: str, **kwargs) -> Tuple[str, int]:
             if value is not None
         }
     )
-    # returns message_id, which we don't use
-    future.result()
+    try:
+        # publish message
+        client.publish(**publish_kwargs).result()
+    except Exception:  # TODO only catch transient exceptions
```

Comment:
I think your TODO already covers this, but I want to be sure: `except Exception` almost always masks unexpected problems. Catching more specific errors is generally a better idea.

Reply:
i am absolutely not going to leave this catching `Exception`, I just haven't figured out what the block throws yet.